yt_dlp/compat/urllib/request.py

```diff
@@ -0,0 +1,36 @@
+# flake8: noqa: F405
+from urllib.request import *  # noqa: F403
+from .. import compat_os_name
+from ..compat_utils import passthrough_module
+import sys
+
+passthrough_module(__name__, 'urllib.request')
+del passthrough_module
```

Comment:
```suggestion
import sys

from .. import compat_os_name
from ..compat_utils import passthrough_module

passthrough_module(__name__, 'urllib.request')
del passthrough_module
```
Same and maybe reordered